### PR TITLE
API 6503 Location Organization Search

### PR DESCRIPTION
**Pull request by that-guy-is-it**
Updated Location schema to include Managing Organization.

@nmoisan @jtaylor-va @haidong pinging as requested.

### CODE REVIEW

File: db/migration/V15_1__alter_location.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE [App].[Location] ADD [ManagingOrganization] INT NULL
```
Comment (jblefkove-va) [inline db/migration/V15_1__alter_location.sql:1]: Two columns: https://github.com/department-of-veterans-affairs/health-apis-data/blob/master/OIT_Lighthouse/Views/Location.sql

File: db/migration/V15_1__alter_location.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE [App].[Location] ADD [ManagingOrgIdNumber] INT NOT NULL, [ManagingOrgResourceCode] [char](1) NOT NULL
```
Comment (jblefkove-va) [inline db/migration/V15_1__alter_location.sql:1]: these have to be `NULL` so we can add the columns without populating them

Comment (jtaylor-va) [reply db/migration/V15_1__alter_location.sql]: How does this data get populated? You could always alter the columns back to `NOT NULL` once the data has been inserted into the columns.

Comment (that-guy-is-it) [reply db/migration/V15_1__alter_location.sql]: @jtaylor-va This data will get populated in a future PR. It is dependent on a DQ Entity update, so it cannot e populated now. The columns _could_ be reverted to `NOT NULL` after population, but for this test data we can tolerate the discrepancy and have done so in the past.

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**